extensions/roam-research/src/roamApi.ts

```diff
@@ -1,139 +1,140 @@
-import { spawn } from "child_process";
 import dayjs from "dayjs";
-import { parseTemplate } from "./parse-template";
-import type { ReversePullBlock } from "./type";
+import { parseTemplate, tagPageTitlesStrSuffix } from "./parse-template";
 import { todayUid } from "./utils";
+import * as roamApiSdk from "./roam-api-sdk-copy";
+import { Cache } from "@raycast/api";
 
-const api =
-  (graphName: string, graphToken: string) =>
-  <T>(operationName: string) =>
-  (query: string) =>
-    new Promise<RoamResponse<T>>((resolve, reject) => {
-      const spawnProcess = spawn("curl", [
```

Comment:
previously we used to spawn a curl process for each backend request. Now we use `roam-api-sdk-copy.ts` which uses `fetch` from `cross-fetch` (used this dep because it's used by Raycast itself, for example, in its `useFetch` hook)